R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -3990,12 +3990,21 @@ test_that("Call DataFrameWriter.load() API in Java without path and check argume
   expect_error(read.df(source = "json"),
                paste("Error in load : analysis error - Unable to infer schema for JSON.",
                      "It must be specified manually"))
-  expect_error(read.df("arbitrary_path"), "Error in load : analysis error - Path does not exist")
-  expect_error(read.json("arbitrary_path"), "Error in json : analysis error - Path does not exist")
-  expect_error(read.text("arbitrary_path"), "Error in text : analysis error - Path does not exist")
-  expect_error(read.orc("arbitrary_path"), "Error in orc : analysis error - Path does not exist")
+  expect_error(read.df("arbitrary_path"),
```

Comment:
Is there a way to do payload-only testing in R instead of matching on the error text? This approach is brittle and will cause breaks when we refine the message text.

Reply:
Yeah, I totally agree that the current approach too easy to break so we'd better to find nicer way, but I have no idea to test SparkR in my test environment for now.

Let me manually fix the text first, while figuring out a better way to test it.

Reply:
@itholic Could you open an JIRA to not forget about this, please.